master/buildbot/test/unit/test_worker_ec2.py

```diff
@@ -572,6 +581,10 @@ def botoSetup(self):
         except NotImplementedError:
             raise unittest.SkipTest("KeyPairs.create_key_pair not implemented"
                                     " in this version of moto, please update.")
+        except (botocore.exceptions.ClientError,
```

Comment:
I think unit tests should not require EC2 credentials. As those are fully mocked by moto we shouln't get a credential error